examples/undocumented/python_modular/graphical/classifier_gaussian_process_binary_classification.py

```diff
@@ -7,25 +7,27 @@
 # Written (W) 2013 Roman Votyakov
 #
 
-from pylab import *
-from numpy import *
-from itertools import *
+import matplotlib
+matplotlib.use('Agg')   # Comment this to see the plot
```

Comment:
this should not be in the code file. It should only be put there when the script is called from the cmake test